wiki/release/release-process.md

```diff
@@ -19,42 +19,48 @@
 
 -->
 
-# Release Process
+* [Release Process](#release-process)
+   * [1. Create the release branch](#1-create-the-release-branch)
+   * [2. Update project version and tag](#2-update-project-version-and-tag)
+   * [3. Build and inspect the artifacts](#3-build-and-inspect-the-artifacts)
+      * [3.1. Build RPM and DEB packages](#31-build-rpm-and-deb-packages)
+   * [4. Sign and stage the artifacts](#4-sign-and-stage-the-artifacts)
+   * [5. Stage artifacts in maven](#5-stage-artifacts-in-maven)
+   * [6. Run the vote](#6-run-the-vote)
+   * [7. Move master branch to next version](#7-move-master-branch-to-next-version)
+   * [8. Promote the release](#8-promote-the-release)
+   * [9. Publish Docker Images](#9-publish-docker-images)
+      * [A. No push access to apachepulsar org - use Personal account for staging the Docker images](#a-no-push-access-to-apachepulsar-org---use-personal-account-for-staging-the-docker-images)
+         * [Publish Docker Images to personal account in hub.docker.com (if you don't have access to apachepulsar org)](#publish-docker-images-to-personal-account-in-hubdockercom-if-you-dont-have-access-to-apachepulsar-org)
+         * [Copy Docker image from the Release manager's personal account to apachepulsar org](#copy-docker-image-from-the-release-managers-personal-account-to-apachepulsar-org)
+      * [B. Push access to apachepulsar org](#b-push-access-to-apachepulsar-org)
+         * [Publish Docker Images directly to apachepulsar org in hub.docker.com](#publish-docker-images-directly-to-apachepulsar-org-in-hubdockercom)
+   * [10. Release Helm Chart](#10-release-helm-chart)
+   * [11. Publish Python Clients](#11-publish-python-clients)
+      * [Linux](#linux)
+      * [MacOS](#macos)
+   * [12. Publish MacOS libpulsar package](#12-publish-macos-libpulsar-package)
+   * [13. Update Python Client docs](#13-update-python-client-docs)
+   * [14. Update swagger file](#14-update-swagger-file)
+   * [15. Write release notes](#15-write-release-notes)
+   * [16. Update the site](#16-update-the-site)
+      * [Update the site for major releases](#update-the-site-for-major-releases)
+      * [Update the site for minor releases](#update-the-site-for-minor-releases)
+   * [17. Announce the release](#17-announce-the-release)
+   * [18. Write a blog post for the release (optional)](#18-write-a-blog-post-for-the-release-optional)
+   * [19. Remove old releases](#19-remove-old-releases)
+   * [20. Move release branch to next version](#20-move-release-branch-to-next-version)
+
```

Comment:
We don't need this manually generated toc. On GitHub you can read toc as:

<img width="1728" alt="image" src="https://user-images.githubusercontent.com/18818196/188556629-6759b7aa-6acb-44c9-b9ca-05b9a9a2a56f.png">

A manual toc is hard to maintain.

Reply:
It makes sense. I think we can remove the number as well. Otherwise, when you insert a section, you have to change all subsequent section numbers.